.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,5 +17,7 @@ https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#pull-requ
 | Deprecations? | yes / no
 | Fixed ticket? | Fixes #{issue number here}.
 | How to test?  | Please indicate how to best verify that this PR is correct.
+| What are the parts of PrestaShop are modified this PR?  | Please indicate what parts of the software we need to verify to check everything is allright
```

Comment:
```suggestion
| What are the parts of PrestaShop modified by this PR?  | Please indicate what parts of the software we need to verify to check everything is alright
```

Reply:
Waoh I am really tired... I write such bad English 😅 . Thanks for proofreading!